packages/rest/test/unit/request-body.validator.test.ts

```diff
@@ -56,8 +68,17 @@ describe('validateRequestBody', () => {
   });
 
   it('rejects data containing values of a wrong type', () => {
+    const details = [
```

Comment:
Please consider adding an explicit type information for `details` so that the compiler can verify whether the property names below are matching the interface. Same comment applies to other test cases below too.

E.g.

```ts
const details: ErrorDetails[] = [
  // ...
];
```